mindmaps-graph/src/main/java/io/mindmaps/graph/internal/AbstractMindmapsGraph.java

```diff
@@ -158,8 +155,7 @@ public G getTinkerPopGraph(){
 
     @Override
     public GraphTraversalSource getTinkerTraversal(){
-        ReadOnlyStrategy readOnlyStrategy = ReadOnlyStrategy.instance();
-        return getTinkerPopGraph().traversal().asBuilder().with(readOnlyStrategy).create(getTinkerPopGraph());
+        return getTinkerPopGraph().traversal();
```

Comment:
It's a conflict between OrientDb using 3.2 and Titan using 3.1. The former uses `traversal.withStratergies()`, the latter uses `traversal.asBuilder().with()`